app-i18n/rime-data/rime-data-0.0.0.20230204.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+MY_PN="rime-essay"
+MY_COMMIT="e0519d0579722a0871efb68189272cba61a7350b"
+
+DESCRIPTION="Data resources for Rime Input Method Engine"
+HOMEPAGE="https://rime.im/ https://github.com/rime/plum"
+SRC_URI="https://github.com/rime/${MY_PN}/archive/${MY_COMMIT}.tar.gz -> ${MY_PN}-${PV}.tar.gz"
+
+LICENSE="LGPL-3"
+SLOT="essay"
+KEYWORDS="~amd64 ~arm64 ~loong ~ppc ~ppc64 ~riscv ~x86"
+IUSE="
+	array
+	+bopomofo
+	+cangjie
+	combo-pinyin
+	double-pinyin
+	emoji
+	ipa
+	+luna-pinyin
+	middle-chinese
+	pinyin-simp
+	quick
+	scj
+	soutzoe
+	stenotype
+	+stroke
+	+terra-pinyin
+	wubi
+	wugniu
+"
+
+DEPEND="!app-i18n/rime-data:0"
+RDEPEND="${DEPEND}"
+PDEPEND="
```

Comment:
Why these are `PDEPEND`? Look like `RDEPEND` to me.

Reply:
Please refer to https://devmanual.gentoo.org/general-concepts/dependencies/#post-dependencies

All these data packages except essay itself and prelude are not necessary for the running of rime. So I will put prelude into RDEPEND and keep others unchanged. Thanks!

Reply:
Can we make it build against system headers?
That is `dev-libs/darts` `dev-libs/utfcpp` `x11-libs/xcb-util-keysyms`.
If not, we should include LICENSE for corresponding headers.